archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -217,6 +217,9 @@ package_clang-git() {
 
     # Move analyzer scripts out of /usr/libexec
     mv "$pkgdir"/usr/libexec/{ccc,c++}-analyzer "$pkgdir/usr/lib/clang/"
+    mv "$pkgdir"/usr/libexec/analyze-{c++,cc} "$pkgdir/usr/lib/clang/"
+    mv "$pkgdir"/usr/libexec/intercept-{c++,cc} "$pkgdir/usr/lib/clang/"
+
     rmdir "$pkgdir/usr/libexec"
```

Comment:
Given this, should this just move everything under `libexec` over?

Reply:
Yeah, i guess?